### PR TITLE
[PLAT-1153] - Use 0.15.x @vertex/viewer

### PR DESCRIPTION
**Pull request by gbiv**
## Summary

Switch the template to use the `@ .15.x` version instead of a pinned version.

## Test Plan

 generate an app of a scene using the scene:render command

## Dependencies

[Ticket](https://vertexvis.atlassian.net/browse/PLAT-1153)

### CODE REVIEW

**Review by amvertex — APPROVED** [review]

File: src/lib/renderer.ts [hunk 1]
```diff
@@ -57,10 +57,18 @@ export function generateHtml(
     <meta charset="utf-8" />
     <title>Getting Started with Vertex</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link
+   <link
       rel="stylesheet"
-      href="https://unpkg.com/@vertexvis/viewer@0.11.x/dist/viewer/viewer.css"
+      href="https://unpkg.com/@vertexvis/viewer@latest/dist/viewer/viewer.css"
```
Comment (danschultz) [inline src/lib/renderer.ts:62]: We pinned this on purpose, so people's demos don't break. Maybe it makes sense to have an CLI option to configure this.

Comment (danschultz) [reply src/lib/renderer.ts:62]: I'd still pin even with the CLI option, e.g. `@latest` is opt-in.

Comment (gbiv) [reply src/lib/renderer.ts:62]: I'm good with pinning it

File: src/lib/renderer.ts [hunk 2]
```diff
@@ -57,10 +57,18 @@ export function generateHtml(
     <meta charset="utf-8" />
     <title>Getting Started with Vertex</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link
+   <link
       rel="stylesheet"
-      href="https://unpkg.com/@vertexvis/viewer@0.11.x/dist/viewer/viewer.css"
+      href="https://unpkg.com/@vertexvis/viewer@0.15.0/dist/viewer/viewer.css"
```
Comment (danschultz) [inline src/lib/renderer.ts:62]: Should be `0.15.x`

File: src/lib/renderer.ts [hunk 3]
```diff
@@ -57,10 +57,18 @@ export function generateHtml(
...
     />
+    <script
+      type="module"
+      src="https://unpkg.com/@vertexvis/viewer@0.15.0/dist/viewer/viewer.esm.js"
+    ></script>
+    <script
+      nomodule
+      src="https://unpkg.com/@vertexvis/viewer@0.15.0/dist/viewer.js"
+    ></script>
```
Comment (danschultz) [inline src/lib/renderer.ts:71]: ^

File: README.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ $ npm install -g @vertexvis/cli
 $ vertex COMMAND
 running command...
 $ vertex (-v|--version|version)
-@vertexvis/cli/0.16.1 darwin-x64 node-v16.14.0
+@vertexvis/cli/0.16.2 darwin-x64 node-v14.17.3
```
Comment (amvertex) [inline README.md:42]: Why are we using an older version of node?

Comment (gbiv) [reply README.md:42]: 😱  How did that get in there???

Comment (gbiv) [reply README.md:42]: oh, this README.md pulls this from my local version

Comment (gbiv) [reply README.md]: @amvertex updated my local node version to the latest lts and re-ran.

**Review by amvertex — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (amvertex) [reply README.md]: Perfect!

**Pull request merged**